gramex/handlers/mlhandler.py

```diff
@@ -208,7 +207,8 @@ def _predict(self, data=None, score_col=''):
     def _check_model_path(self):
         try:
             klass, wrapper = ml.search_modelclass(self.store.load('class'))
-            self.model = locate(wrapper).from_disk(self.store.model_path, klass=klass)
+            if hasattr(self.store, 'model_path') and not op.isdir(self.store.model_path):
+                self.model = locate(wrapper).from_disk(self.store.model_path, klass=klass)
```

Comment:
Hmm... ModelStore needs a better API - this condition looks too complex and unreadable. There should be some property of the store itself which tells you whatever you want to know.

Anyway... why is this required?

And you can write this simply as:
```python
if op.isdir(getattr(self.store, 'model_path', '')):
    do_stuff()
```